webapp/fmprod/src/main/java/org/apache/oodt/cas/product/jaxrs/exceptions/BadRequestException.java

```diff
@@ -13,33 +13,29 @@
  * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
  * See the License for the specific language governing permissions and
  * limitations under the License.
-*/
+ */
 
 package org.apache.oodt.cas.product.jaxrs.exceptions;
 
 import javax.ws.rs.WebApplicationException;
-import javax.ws.rs.core.MediaType;
-import javax.ws.rs.core.Response;
 
 /**
- * This type of exception returns an HTTP 'bad request' response (status code
- * 400) with an additional message.
+ * This type of exception returns an HTTP 'bad request' response (status code 400) with an
+ * additional message.
+ *
  * @author rlaidlaw
  * @version $Revision$
  */
-public class BadRequestException extends WebApplicationException
-{
-  // Auto-generated ID for serialization.
+public class BadRequestException extends WebApplicationException {
```

Comment:
Could you check if this class actually has your changes? If no, let's not format such classes.

Reply:
Yes, I did some changes to the class. (Changed its constructor a bit)

Reply:
Great. I will merge this once I check this on an IDE.